HLTrigger/Configuration/python/CustomConfigs.py

```diff
@@ -31,6 +31,8 @@ def Base(process):
 #        process.GlobalTag.connect   = 'frontier://FrontierProd/CMS_CONDITIONS'
 #        process.GlobalTag.pfnPrefix = cms.untracked.string('frontier://Frontie#rProd/')
 #        
+    process.GlobalTag.snapshotTime = cms.string("9999-12-31 23:59:59.000")
```

Comment:
I think this is not used in production workflows, as it would need to be explcitely called as customisation function.